src/components/dropdown/_dropdown.scss

```diff
@@ -92,7 +92,7 @@
     position: absolute;
     z-index: z('dropdown');
     max-height: 0;
-    transition: max-height $transition--expansion $carbon--ease-out;
+    transition: max-height $carbon--ease-in $transition--expansion $carbon--ease-out;
```

Comment:
Seems that the property value is invalid according to https://developer.mozilla.org/en-US/docs/Web/CSS/transition#Formal_syntax - I no longer see the animation?

Reply:
yes, and by increasing the speed, the animation disappears, the issue is caused by the property `overflow: auto`  putting this property to `overflow: scroll` will solve the problem but the scroll bar will still be displayed, If the expanded height doesn't need a scrollbar it will be inactive

Reply:
So you mean that your intent is killing the animation?

Reply:
no, just find a fix without visual style change.

Reply:
Thanks @rezak-otmani for clarifying - Seems that your fix actually is killing the animation:

![invalid-css-prop](https://user-images.githubusercontent.com/1259051/38626012-53b39ac0-3d71-11e8-9f77-0b6fa2a40db7.png)

Would you want to fix it...? Thanks!